resources/assets/less/photoswipe-overrides.less

```diff
@@ -28,6 +28,16 @@
       color: @contest-selected-colour;
     }
 
+    &--vote-progress {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      && {
```

Comment:
sorry, I just noticed the element has 0.75 opacity on normal state and has pointer cursor on hover which is weird because it's not clickable and shouldn't have hover effect.

Reply:
No problem.